lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorRequestBuilder.cs

```diff
@@ -95,7 +103,18 @@ public EvaluatorRequestBuilder SetRackName(string rackName)
         }
 
         /// <summary>
-        /// Sets the batch ID for requested evaluators in the same request. The batch of Evaluators requested in the 
+        /// Set desired node names for the Evaluator to be allocated on.
+        /// </summary>
+        /// <param name="nodeNames"></param>
+        /// <returns>this</returns>
+        public EvaluatorRequestBuilder SetNodeNames(IList<string> nodeNames)
```

Comment:
This is tricky if the method is called more than once, right? Wouldn't it be cleaner to have `AddNodeName(string)` and `AddNodeNames(IEnumerable<string>)`?